docs/source/community/host-guide.md

```diff
@@ -0,0 +1,126 @@
+
+Partner with another contributor (a meeting facilitator) to do the following:
+
+- [ ] Log in as the “Project Jupyter” host account for meeting management perks (removing spammers, muting and recording, etc.), [request access](mailto:security@ipython.org) from the Jupyter Security Team.
+- [ ] Record the meeting, if that is intended. The host starts and stops the recording. These are posted to YouTube.
+- [ ] Remove unwelcome recording bots:
+  - Per the Jupyter community page, participants
+  should not record meetings on their own. The host should ask participants
+  that look like AI recording bots:
+  - To identify themselves and state whether they're recording
+  - If they get no response from the account, the host should assume
+    it's recording, and remove it from the meeting.
+## &#x270f; Suggested Meeting Script and Checklist 
+### &#x23F3; Prior to meeting start
+
+- [ ] Log in to Zoom from the Project Jupyter account [(*request access*](mailto:security@ipython.org). Note this is necessary in order to record and manage meeting participants.  Alternatively, provided that someone who has host privileges logs onto the call they can grant host privileges in real time at the start of the meeting. 
+*from the Jupyter Security group if needed*)
+- [ ] Make sure you have a stable internet
+connection, a good quality microphone, and a quiet environment.
+- [ ] Check the [meeting agenda](insert agenda)
+  - *The agenda is a collaboratively created document made by the participants.*
+    *Ensure a section for today's meeting is created, with today's date and a table*
+    *for participants to record their names (which can be copied/pasted from lower*
+    *entries). The agenda will also be used as meeting minutes.*
+
+&#x1F517; *Links to copy/paste into the meeting chat:*
+
+- Weekly Meeting Notes: [Insert your team’s notes here]
+- Jupyter Code of Conduct: https://jupyter.org/governance/conduct/code_of_conduct.html
+- [Subproject Name]Team Compass: 
+
+### &#x1F399; Start the meeting! 
+
+```
+Hello and welcome to our [day, month, year] Jupyter [TOPIC_HERE] call. I’m
+[host name] and I’ll be your host today.
+This is a place to for contributors to connect with each other
+and the community about [TOPIC_HERE] topics. A special highlight and welcome to all first time participants! We want all newcomers to feel
+welcome, we invite you to join in on discussions, introduce yourself,
+or add items to the agenda.
+
+Please keep in mind that this call will be recorded and posted to YouTube for the community to view. This call is a part of the Jupyter community, therefore we follow Jupyter Code of Conduct, which you can read about at jupyter.org/conduct[Meeting host insert link in the meeting minutes template]
+
+Before we begin, I want to share how topics make it on to the agenda. Presenters have submitted what they want to discuss or share ahead of time. We’ll have a little time for discussion with each one.
+```
```

Comment:
```suggestion
> Hello and welcome to our [day, month, year] Jupyter [TOPIC_HERE] call. I’m
> [host name] and I’ll be your host today.
> This is a place to for contributors to connect with each other
> and the community about [TOPIC_HERE] topics. A special highlight and welcome to all first time > participants! We want all newcomers to feel
> welcome, we invite you to join in on discussions, introduce yourself,
> or add items to the agenda.
> 
> Please keep in mind that this call will be recorded and posted to YouTube for the community to view. This call is a part of the Jupyter community, therefore we follow Jupyter Code of Conduct, which you can read about at jupyter.org/conduct[Meeting host insert link in the meeting minutes template]
> 
> Before we begin, I want to share how topics make it on to the agenda. Presenters have submitted what they want to discuss or share ahead of time. We’ll have a little time for discussion with each one.
```

The `> ` syntax makes the text appear as a blockquote, not as formatted source code.